packages/react-components/react-infobutton/src/components/InfoButton/useInfoButtonStyles.ts

```diff
@@ -51,54 +50,102 @@ const useButtonStyles = makeStyles({
         display: 'none',
       },
     },
-    ':enabled:hover:active': {
+    ':hover:active': {
       backgroundColor: tokens.colorTransparentBackgroundPressed,
       color: tokens.colorNeutralForeground2BrandPressed,
     },
-    ':disabled': {
-      cursor: 'not-allowed',
-      color: tokens.colorNeutralForegroundDisabled,
+  },
+
+  selected: {
+    backgroundColor: tokens.colorTransparentBackgroundSelected,
+    color: tokens.colorNeutralForeground2BrandSelected,
+
+    [`& .${iconFilledClassName}`]: {
+      display: 'inline-flex',
+    },
+    [`& .${iconRegularClassName}`]: {
+      display: 'none',
+    },
+
+    '@media (forced-colors: active)': {
+      backgroundColor: 'Highlight',
+      ...shorthands.borderColor('Canvas'),
+      color: 'Canvas',
+    },
+  },
+
+  highContrast: {
+    '@media (forced-colors: active)': {
+      ...shorthands.borderColor('Canvas'),
+      color: 'CanvasText',
+
+      ':hover': {
+        forcedColorAdjust: 'none',
+        backgroundColor: 'Highlight',
+        ...shorthands.borderColor('Canvas'),
+        color: 'Canvas',
+      },
+
+      ':hover:active': {
+        forcedColorAdjust: 'none',
+        backgroundColor: 'Highlight',
+        ...shorthands.borderColor('Canvas'),
+        color: 'Canvas',
+      },
     },
   },
 
   focusIndicator: createCustomFocusIndicatorStyle({
-    ...shorthands.borderRadius(tokens.borderRadiusSmall),
+    ...shorthands.borderRadius(tokens.borderRadiusMedium),
     ...shorthands.borderColor(tokens.colorTransparentStroke),
     outlineColor: tokens.colorTransparentStroke,
     outlineWidth: tokens.strokeWidthThick,
     outlineStyle: 'solid',
     boxShadow: `
       ${tokens.shadow4},
-      0 0 0 ${tokens.borderRadiusSmall} ${tokens.colorStrokeFocus2}
+      0 0 0 2px ${tokens.colorStrokeFocus2}
     `,
     zIndex: 1,
   }),
 
-  selected: {
-    backgroundColor: tokens.colorTransparentBackgroundSelected,
-    color: tokens.colorNeutralForeground2BrandSelected,
-
-    [`& .${iconFilledClassName}`]: {
-      display: 'inline-flex',
-    },
-    [`& .${iconRegularClassName}`]: {
-      display: 'none',
-    },
+  small: {
+    ...shorthands.padding(tokens.spacingVerticalXS, tokens.spacingHorizontalXS),
+  },
+  medium: {
+    ...shorthands.padding(tokens.spacingVerticalXS, tokens.spacingHorizontalXS),
+  },
+  large: {
+    ...shorthands.padding(tokens.spacingVerticalXXS, tokens.spacingVerticalXXS),
   },
 });
 
+const usePopoverSurfaceStyles = makeStyles({
+  small: typographyStyles.caption1,
+  medium: typographyStyles.caption1,
```

Comment:
Yes it's a trade-off, but in this case I think it's worth changing, since two out of three of these styles are identical, and `typographyStyles` brings in a bunch of styles.

One option to further simplify it is to _only_ have a `smallMedium` style, applied when `size !=== 'large'`, because the default typography style is `body1`, and this is the root of the surface.